lib/model_patches.rb

```diff
@@ -8,6 +8,18 @@
 Rails.configuration.to_prepare do
 
   # Remove UK-specific references to FOI
+  InfoRequest.instance_eval do
+
+    SITE_LAW_USED_READABLE_DATA =
```

Comment:
You'll also need to [generate the theme translations](https://github.com/mysociety/alaveteli/wiki/Generate-theme-translations) given that you've added new `msgid`s. Remember to [update theme translations](https://github.com/mysociety/alaveteli/wiki/Update-theme-translations) before pushing.